resolwe_bio/processes/import_data/nucleotide_seq.yml

```diff
@@ -7,6 +7,9 @@
   name: Upload nucleotide sequence(s) (FASTA)
   requirements:
     expression-engine: jinja
+    executor:
+      docker:
+        image: "resolwe/bio-linux8-resolwe-preview:icount"
```

Comment:
Test that will pull docker image "icount" are:

- test_icount
- test_reads_filtering.ReadsFilteringProcessorTestCase.test_sortmerna_single
- test_reads_filtering.ReadsFilteringProcessorTestCase.test_sortmerna_paired